x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/store/SearchableSnapshotDirectoryTests.java

```diff
@@ -566,8 +566,7 @@ private void testDirectories(
             }
 
             final Store store = new Store(shardId, indexSettings, directory, new DummyShardLock(shardId));
-            store.incRef();
-            releasables.add(store::decRef);
+            releasables.add(store::close);
```

Comment:
I'm not sure why the lifecycle was being managed like above, and whether I've fundamentally changed something here. It seems like we should have a single reference from creation, so we don't need to `incRef`?